docs/presentations/linkedin-meetup-2024-04-24.qmd

```diff
@@ -0,0 +1,549 @@
+
+## DataFrame history
+
+- **pandas** (2008): dataframes in Python
+- **Spark** (2009): distributed dataframes with PySpark
+- **Dask** (2014): distributed dataframes with Python
+- **dplyr** (2014): dataframes in R with SQL-like syntax
+- **Ibis** (2015): dataframes in Python with SQL-like syntax
+- **cuDF** (2017): pandas on GPUs
+- **Modin** (2018): pandas on Ray/Dask
+- **Koalas** (2019): pandas on Spark
+- **Polars** (2020): multicore dataframes in Python
+
+## Two world problem {auto-animate="true"}
+
+::: {.nonincremental}
+:::: {.columns}
+
+::: {.column}
+SQL:
+:::
+
+::: {.column}
+Python:
+:::
+
+::::
+:::
+
+## Two world problem {auto-animate="true"}
+
+::: {.nonincremental}
+:::: {.columns}
+
+::: {.column}
+SQL:
+
+- databases & tables
+:::
+
+::: {.column}
+Python:
+
+- files & dataframes
+:::
+
+::::
+:::
+
+## Two world problem {auto-animate="true"}
+
+::: {.nonincremental}
+:::: {.columns}
+
+::: {.column}
+SQL:
+
+- databases & tables
+- analytics
+:::
+
+::: {.column}
+Python:
+
+- files & dataframes
+- data science
+:::
+
+::::
+:::
+
+## Two world problem {auto-animate="true"}
+
+::: {.nonincremental}
+:::: {.columns}
+
+::: {.column}
+SQL:
+
+- databases & tables
+- analytics
+- metrics
+:::
+
+::: {.column}
+Python:
+
+- files & dataframes
+- data science
+- statistics
+:::
+
+::::
+:::
+
+## Two world problem {auto-animate="true"}
+
+::: {.nonincremental}
+:::: {.columns}
+
+::: {.column}
+SQL:
+
+- databases & tables
+- analytics
+- metrics
+- dashboards
+:::
+
+::: {.column}
+Python:
+
+- files & dataframes
+- data science
+- statistics
+- notebooks
+:::
+
+::::
+:::
+
+## Two world problem {auto-animate="true"}
+
+::: {.nonincremental}
+:::: {.columns}
+
+::: {.column}
+SQL:
+
+- databases & tables
+- analytics
+- metrics
+- dashboards
+:::
+
+::: {.column}
+Python:
+
+- files & dataframes
+- data science
+- statistics
+- notebooks
+:::
+
+::::
+:::
+
+::: {.r-fit-text}
+***Ibis bridges the gap.***
+:::
+
+## Bridging the gap
+
+::: {.panel-tabset}
+## DuckDB
+
+```python
+import ibis
+
+con = ibis.duckdb.connect()
+penguins = con.table("penguins")
+penguins.group_by("species", "island").agg(count=ibis._.count())
+```
+
+An embeddable, zero-dependency, C++ SQL database engine.
+
+## DataFusion {auto-animate="true"}
+
+```python
+import ibis
+
+con = ibis.datafusion.connect()
+penguins = con.table("penguins")
+penguins.group_by("species", "island").agg(count=ibis._.count())
+```
+
+A Rust SQL query engine.
+
+## ClickHouse {auto-animate="true"}
+
+```python
+import ibis
+
+con = ibis.clickhouse.connect()
+penguins = con.table("penguins")
+penguins.group_by("species", "island").agg(count=ibis._.count())
+```
+
+A C++ column-oriented database management system.
```

Comment:
these (and the entire tabset) are not executing/showing output

Reply:
Yeah, these won't execute in general because I'd have to spin up a bunch of containers etc.